README.md

```diff
@@ -18,7 +17,19 @@ To get the source code of our SDKs and samples via **git** just type:
     git clone https://github.com/Azure/azure-mobile-services.git
     cd ./azure-mobile-services/
 
-## Change log
+## SDk Documentation references
+
+#### Azure App Service Mobile Apps
+* [iOS](http://azure.github.io/azure-mobile-services/ios/v3)
+* [Managed](http://azure.github.io/azure-mobile-services/managed/v2)
```

Comment:
We won't have a managed, android, or JS any time soon here.  Remove or point them at their current sites (if they exist)